src/sentry/workflow_engine/models/json_config.py

```diff
@@ -15,3 +24,16 @@ def validate_config(self, schema: dict[str, Any]) -> None:
 
     class Meta:
         abstract = True
+
+
+@receiver(pre_save, sender=JSONConfigBase)
```

Comment:
do subclasses of `JSONConfigBase` also trigger this receiver? the existing models with json configs don't all store the schema in the same place so i've made a receiver for each model that subclasses `JSONConfigBase`

Reply:
not sure - i'm adding a test now to see if this works 🤞 (i was hoping reviews would say one way or the other in the mean time 🤣) -- if not i'll just move this over to action.

Reply:
Workflow and Detector differ in where they store the schema (Workflow has a `config` property and Detector looks at `group_type.detector_config_schema`), so the receivers are separate there